server/mediaserver.go

```diff
@@ -137,11 +137,21 @@ type authWebhookResponse struct {
 }
 
 func NewLivepeerServer(rtmpAddr string, lpNode *core.LivepeerNode, httpIngest bool, transcodingOptions string) (*LivepeerServer, error) {
+	var nodeEthAddr string = "n/a"
+	if lpNode.Eth != nil {
+		nodeEthAddr = lpNode.Eth.Account().Address.String()
+	}
 	opts := lpmscore.LPMSOpts{
 		RtmpAddr:     rtmpAddr,
 		RtmpDisabled: true,
 		WorkDir:      lpNode.WorkDir,
 		HttpMux:      http.NewServeMux(),
+		HttpDebug:    vidplayer.VidPlayerDebug {
+			DebugEnabled:    true,
+			LivepeerVersion: core.LivepeerVersion,
+			NodeType:        lpNode.NodeType.String(), 
+			NodeEthAddress:  nodeEthAddr, 
+		},
 	}
```

Comment:
Here's an example of what the headers look like in each config: 

**off-chain:**
<img width="393" alt="Screen Shot 2022-06-07 at 1 05 47 PM" src="https://user-images.githubusercontent.com/3213391/172477542-090e8f4a-a80a-4ddf-94bf-ae6f3ce4061c.png">

**on-chain (using devtool):**
<img width="444" alt="Screen Shot 2022-06-07 at 1 07 14 PM" src="https://user-images.githubusercontent.com/3213391/172477545-01b276ab-4b60-432d-b8f3-71973a4eb183.png">

Is it OK to list the Node's Eth Address in the response headers? It looks like public info to me but please confirm.